operations/mimir-mixin/docs/playbooks.md

```diff
@@ -1102,6 +1102,98 @@ An exemplar must have a valid timestamp, otherwise it cannot be correlated to an
 
 > **Note**: Invalid exemplars are skipped during the ingestion, and valid exemplars within the same request are ingested.
 
+### err-mimir-ingester-max-ingestion-rate
+
+This critical error occurs when the rate of received samples per second is exceeded in an ingester.
+
+The ingester implements a rate limit on the samples per second that can be ingested, and it's used to protect an ingester from overloading in case of an high traffic.
```

Comment:
```suggestion
The ingester implements a rate limit on the samples per second that can be ingested, and it's used to protect an ingester from overloading in case of high traffic.
```